static/src/stylesheets/module/facia/_items.scss

```diff
@@ -49,6 +52,10 @@
     }
 }
 .fc-slice__item--no-mpu {
+    -webkit-box-flex: 0 !important;
+    -moz-box-flex: 0 !important;
+    -webkit-flex: 0 !important;
+    -ms-flex: 0 !important;
```

Comment:
Pretty sure we already have a helper for this [here](https://github.com/guardian/guss-css3/blob/master/_css3.scss#L133)